### PR TITLE
HeMIS loader

### PR DESCRIPTION
**Pull request by andreanne-lemay**
Modification for HeMIS:
- Don't normalize empty slices or images since it generated NaN
- Stack gt in ToTensor instead than in StackTensors. StackTensors is now only needed for input multichannel

### CODE REVIEW

File: medicaltorch/transforms.py [hunk 1]
```diff
@@ -323,11 +329,13 @@ def __call__(self, sample):
         input_data = sample['input']
         if isinstance(input_data, list):
             for i in range(len(input_data)):
-                mean, std = input_data[i].mean(), input_data[i].std()
-                input_data[i] = F.normalize(input_data[i], [mean], [std])
+                if input_data[i].type(torch.bool).any():
+                    mean, std = input_data[i].mean(), input_data[i].std()
+                    input_data[i] = F.normalize(input_data[i], [mean], [std])
```
Comment (charleygros) [inline medicaltorch/transforms.py:334]: The empty slices should be filtered at this point... or am I missing something?

**Review by charleygros — APPROVED** [review]
